website/guides/terraform-provider-development-program.html.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: "guides"
+page_title: "Terraform Provider Development Program"
+sidebar_current: "guides-terraform-provider-development-program"
+description: This guide is intended for vendors who're interested in having their platform supported by Teraform. The guide walks vendors through the steps involved in creating a provider and applying for it to be included with Terraform.
+---
+
+# Terraform Provider Development Program
+
+The Terraform Provider Development Program allows vendors to build
+Terraform providers that are officially approved and tested by HashiCorp and
+listed on the official Terraform website. The program is intended to be largely
+self-serve, with links to information sources, clearly defined steps, and
+checkpoints.
+
+-> **Building your own provider?** If you're building your own
+provider and aren't interested in having HashiCorp officially maintain
+the provider, refer to the
+[Writing Custom Providers guide](/guides/writing-custom-terraform-providers.html).
+
+## What is a Terraform Provider?
+
+Terraform is used to create, manage, and manipulate infrastructure resources.
+Examples of resources include physical machines, VMs, network switches, containers, etc.
+Almost any infrastructure noun can be represented as a resource in Terraform.
+
+A provider is responsible for understanding API interactions with the underlying
+infrastructure like a cloud (AWS, GCP, Azure), a PaaS service (Heroku), a SaaS
+(service DNSimple, CloudFlare), or on-prem resources (vSphere). It then exposes
+these as resources users can code to. Terraform presently supports more than
+70 providers, a number that has more than doubled in the past 12 months.
+
+All providers integrate into and operate with Terraform exactly the same way.
+The table below is intended to help users understand who develops, maintains
+and tests a particular provider.
+
+![Provider Engagement Table](/assets/images/docs/engage-table.png)
+
+This document is primarily intended for the "HashiCorp/Vendors" row in
```

Comment:
Not a blocker, but can we add some kind of emphasis here? Either make it a `-> Note` or some bolding? I believe a big part of the confusion earlier was not being clear about the audience. We're clear here, but I imagine without any call out here it may be glossed over.